src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="http://localhost:8080/"
+      method="post"
+      data-qa="big"
+      class="big-form"
+    >
+      <img
+        src="images/Search.svg"
+        alt="icon"
+        class="big-icon"
+      >
+      <input
+        type="text"
+        class="input-big"
+        id="big-search-bar"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+      <label for="big-search-bar"></label>
+    </form>
+
+    <form
+      action="http://localhost:8080/"
+      method="post"
+      data-qa="small"
+      class="small-form"
+    >
+      <img
+        src="images/Search.svg"
+        alt="icon"
+        class="small-icon"
+      >
+      <input
+        type="text"
+        class="input-small"
+        id="small-search-bar"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+      <label for="small-search-bar"></label>
```

Comment:
and cover input AND image in label, so we can also click on the image for input focus